contracts/GSN/IRelayHub.sol

```diff
@@ -141,16 +195,25 @@ contract IRelayHub {
         bytes memory approvalData
     ) public;
 
-    // Emitted when an attempt to relay a call failed. This can happen due to incorrect relayCall arguments, or the
-    // recipient not accepting the relayed call. The actual relayed call was not executed, and the recipient not charged.
-    // The reason field contains an error code: values 1-10 correspond to PreconditionCheck entries, and values over 10
-    // are custom recipient error codes returned from acceptRelayedCall.
+    /**
+     * @dev Emitted when an attempt to relay a call failed.
+     *
+     * This can happen due to incorrect {relayCall} arguments, or the recipient not accepting the relayed call. The
+     * actual relayed call was not executed, and the recipient not charged.
+     *
+     * The `reason` field contains an error code: values 1-10 correspond to `PreconditionCheck` entries, and values
+     * over 10 are custom recipient error codes returned from {acceptRelayedCall}.
+     */
     event CanRelayFailed(address indexed relay, address indexed from, address indexed to, bytes4 selector, uint256 reason);
 
-    // Emitted when a transaction is relayed. Note that the actual encoded function might be reverted: this will be
-    // indicated in the status field.
-    // Useful when monitoring a relay's operation and relayed calls to a contract.
-    // Charge is the ether value deducted from the recipient's balance, paid to the relay's owner.
+    /**
+     * @dev Emitted when a transaction is relayed. * Useful when monitoring a relay's operation and relayed calls to a
+     * contract.
+     *
+     * Note that the actual encoded function might be reverted: this is indicated in the `status` field.
```

Comment:
Is it event parameters rather than fields? https://solidity.readthedocs.io/en/develop/abi-spec.html?highlight=parameter#events

```suggestion
     * Note that the actual encoded function might be reverted: this is indicated in the `status` parameter.
```